### ISSUE TITLE
Can I train with rectangular 4:3 images?

### ISSUE THREAD

**Issue by sarmientoj24** [lucidrains/stylegan2-pytorch#168]
Is it possible to train with rectangular images? Where do I set the image size?
My training image is about 800x600 in resolution.

**Comment by lucidrains** [comment]
Not yet, but I'll see what I can do next week! The images will still have to be a power of 2, so maybe 512 x 384, or 1024 x 768

**Comment by sarmientoj24** [comment]
^thank you. hopefully that's possible

**Comment by lucidrains** [comment]
@sarmientoj24 yup, it will be, but the ratios will only come as 1 - 4, 1 - 2, or 3 - 4
